pkg/strategy/supertrend/strategy.go

```diff
@@ -22,13 +23,60 @@ const stateKey = "state-v1"
 
 var log = logrus.WithField("strategy", ID)
 
+// TODO: limit order for ATR TP
+
 func init() {
 	// Register the pointer of the strategy struct,
 	// so that bbgo knows what struct to be used to unmarshal the configs (YAML or JSON)
 	// Note: built-in strategies need to imported manually in the bbgo cmd package.
 	bbgo.RegisterStrategy(ID, &Strategy{})
 }
 
+// LinGre is Linear Regression baseline
+type LinGre struct {
```

Comment:
Move this to a single file?

Reply:
It's not enough to be a standalone indicator
should it be in a single file?